src/igvfd/schemas/changelogs/reference_file.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`reference_file.json`*
 
+### Schema version 16
+
+* Adjust `content_type` enum list to disallow `regulatory_regions` for all users and admins.
+* Extend `content_type` enum list to include `genomic_elements` for admins.
```

Comment:
Don't think you need to say "for admins"

Reply:
I added it since users can't use this property